source/solvers/navier_stokes_vof_assemblers.cc

```diff
@@ -445,14 +450,12 @@ GLSNavierStokesVOFAssemblerBDF<dim>::assemble_rhs(
         velocity[p + 1] = scratch_data.previous_velocity_values[p][q];
 
 
-      std::vector<double> densities(number_of_previous_solutions(method) + 1);
 
       densities[0] = scratch_data.density[q];
 
-      for (unsigned int p = 1; p < number_of_previous_solutions(method) + 1;
-           ++p)
+      for (unsigned int p = 0; p < number_of_previous_solutions(method); ++p)
```

Comment:
yeah good catch, no idea why this loop was different from the others

Reply:
can you change it also on l. 385?

Reply:
Yup will do